doc/source/whatsnew/v1.5.0.rst

```diff
@@ -474,6 +474,7 @@ Performance improvements
 
 Bug fixes
 ~~~~~~~~~
+- Fixed bug in window rolling when centering with axis=1 (:issue:`46135`)
```

Comment:
This should go under the `Groupby/resample/rolling` section

Reply:
Also should mention specifically when `center=True` and `win_type` is specified.